com.rtosta.zapzap.yaml

```
@@ -0,0 +1,36 @@
+app-id: com.rtosta.zapzap
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: "6.2"
+comand: "zapzap"
```

```suggestion
command: "zapzap"
```

There is warning in the build about this non existent property (see typo)

And I'm wondering if that's not the source of all the problem.